lib/dl_api_lib_testing/dl_api_lib_testing/connector/dataset_suite.py

```diff
@@ -33,3 +38,23 @@ def test_remove_connection(
         sync_us_manager.delete(sync_us_manager.get_by_id(saved_connection_id))
         dataset_resp = control_api.load_dataset(saved_dataset)
         assert dataset_resp.status_code == 200, dataset_resp.json
+
+    def test_dataset__delete_result_field__available_after_refresh(
+        self,
+        saved_dataset: Dataset,
+        control_api: SyncHttpDatasetApiV1,
+    ) -> None:
+        # arrange
```

Comment:
nitpick: setup

Reply:
I have already used the arrange-act-assert pattern in this repo. Additionally, there are no usage of setup-execute and arrange-act-assert is accepted way in purest